examples/with-svelte/apps/docs/README.md

```diff
@@ -0,0 +1,38 @@
+# create-svelte
+
+Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+
+## Creating a project
+
+If you're seeing this, you've probably already done this step. Congrats!
+
+```bash
+# create a new project in the current directory
+npm init svelte
+
+# create a new project in my-app
+npm init svelte my-app
+```
+
+## Developing
+
+Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+
+```bash
+npm run dev
+
+# or start the server and open the app in a new browser tab
+npm run dev -- --open
+```
+
+## Building
+
+To create a production version of your app:
+
+```bash
+npm run build
+```
+
+You can preview the production build with `npm run preview`.
```

Comment:
How so? This works from root or the application

```
> npm run preview --workspace apps/docs

> docs@0.0.0 preview
> svelte-kit preview


  SvelteKit v1.0.0-next.355

  local:   http://localhost:3000
  network: not exposed
```

Reply:
I meant for this comment to apply to the whole Developing and Building sections. It's fine, I don't have significant complaints, it's just that we should likely be telling them to think in Turborepo terms instead of Svelte terms. I know it works, but *why* it works is "well, let me tell you about ..."

Reply:
Ah yea I see where you're coming from. This README is from the create svelte starter and I left it largely (entirely?) untouched to keep things consistent across examples (the remix app does this as well)